integration/sawtooth_integration/tests/test_track_and_trade.py

```diff
@@ -602,6 +607,38 @@ def test_track_and_trade(self):
             }
         )
 
+        get_record_property = jin.get_record_property(
+            'fish-456', 'temperature')
+
+        log_json(get_record_property)
+
+        self.assertIn('dataType', get_record_property)
+        self.assertIn('name', get_record_property)
+        self.assertIn('recordId', get_record_property)
+        self.assertEqual(get_record_property['dataType'], 'INT')
+        self.assertEqual(get_record_property['name'], 'temperature')
+        self.assertEqual(get_record_property['recordId'], 'fish-456')
+
+        # Timestamps and public keys differ for each run of the test,
```

Comment:
I would just assert that the timestamps are set - a minimal test that the api is providing the information.